ynr/apps/elections/uk/templates/candidates/about.html

```diff
@@ -9,6 +9,13 @@
   <h1>About this site</h1>
 {% endblock %}
 
+
+Democracy Club Candidates is an open source platform for crowdsourcing information about candidates standing for political office in the UK.
+
+The resulting data is open for anyone to use. This website powers Democracy Club's <a href="https://whocanivotefor.co.uk">WhoCanIVoteFor.co.uk</a>, and the data has been used by The Electoral Commission and many other sources. 
+
+The core data collected by Democracy Club Candidates includes who is standing, what party they’re standing for, their contact details and social media accounts. The software requires that each change is submitted with a source, so that the collected information can be independently checked.
+
```

Comment:
This block of text needs to be inside the `{% block content %}` section just below it, otherwise it will never show on the site!